packages/component-base/src/gestures.d.ts

```diff
@@ -0,0 +1,79 @@
+// tslint:disable:variable-name Describing an API that's defined elsewhere.
```

Comment:
Let's replace this comment with a license header (we might need to preserve the Polymer authors copyright).

Reply:
Done